fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -63,6 +63,24 @@ struct TenantManagementWorkload : TestWorkload {
 		int64_t tenantCount = 0;
 	};
 
+	enum class TenantOperation : int {
+		CREATE_TENANT = 0,
+		DELETE_TENANT,
+		GET_TENANT,
+		LIST_TENANTS,
+		RENAME_TENANT,
+		CONFIGURE_TENANT,
+		GET_TENANT_GROUP,
+		LIST_TENANT_GROUPS,
+		MAX_OP
```

Comment:
Were you intending to add locking to this list?

Reply:
Yes, though I didn't yet, so I only started with the refactoring (I think with the enum it is slightly more readable and the IDE warns me if I forget to test for something).